packages/elements/src/chart/__test__/mock-config.js

```diff
@@ -786,129 +835,130 @@ let uc1 = {
       },
-      {
-        id: 'a-line-2', // optional
-        type: 'line',
-        mode: 'vertical',
-        scaleID: 'x-axis-1',
-        value: '1',
-        borderColor: 'rgba(128, 128, 128, 0.5)',
-        borderWidth: 2,
-        borderDash: [6, 2],
-        topBound: '1',
-        bottomBound: '-1'
-      },
-      {
-        id: 'a-line-3', // optional
-        type: 'line',
-        mode: 'vertical',
-        scaleID: 'x-axis-1',
-        value: '2',
-        borderColor: 'rgba(128, 128, 128, 0.5)',
-        borderWidth: 2,
-        borderDash: [6, 2],
-        topBound: '3',
-        bottomBound: '-2'
-      },
-      {
-        id: 'a-line-4', // optional
-        type: 'line',
-        mode: 'vertical',
-        scaleID: 'x-axis-1',
-        value: '3',
-        borderColor: 'rgba(128, 128, 128, 0.5)',
-        borderWidth: 2,
-        borderDash: [6, 2],
-        topBound: '1',
-        bottomBound: '-0'
+      scales: {
+        y: {
+          position: 'left',
+          title: {
+            display: true,
+            text: 'Trades'
+          }
+        },
+        // xAxes: [{
+        //   id: 'x-axis-1',
+        // }]
       }
-    ]
+    },
+    annotation: {
+      annotations: [{
+          id: 'a-line-1', // optional
+          type: 'line',
+          mode: 'vertical',
+          scaleID: 'x-axis-1',
+          value: '0',
+          borderColor: 'rgba(128, 128, 128, 0.5)',
+          borderWidth: 6,
+          borderDash: [6, 2],
+          topBound: '2',
+          bottomBound: '-2'
+        },
+        {
+          id: 'a-line-2', // optional
+          type: 'line',
+          mode: 'vertical',
+          scaleID: 'x-axis-1',
+          value: '1',
+          borderColor: 'rgba(128, 128, 128, 0.5)',
+          borderWidth: 2,
+          borderDash: [6, 2],
+          topBound: '1',
+          bottomBound: '-1'
+        },
+        {
+          id: 'a-line-3', // optional
+          type: 'line',
+          mode: 'vertical',
+          scaleID: 'x-axis-1',
+          value: '2',
+          borderColor: 'rgba(128, 128, 128, 0.5)',
+          borderWidth: 2,
+          borderDash: [6, 2],
+          topBound: '3',
+          bottomBound: '-2'
+        },
+        {
+          id: 'a-line-4', // optional
+          type: 'line',
+          mode: 'vertical',
+          scaleID: 'x-axis-1',
+          value: '3',
+          borderColor: 'rgba(128, 128, 128, 0.5)',
+          borderWidth: 2,
+          borderDash: [6, 2],
+          topBound: '1',
+          bottomBound: '-0'
+        }
+      ]
+    }
   }
 };
 
 // user use case
-let uc2 = {
-  type: 'bar',
-  options: {
-    animation: {
-      duration: 0
-    },
-    maintainAspectRatio: false,
-    tooltips: {
-      callbacks: {
+const uc2 = () => {
+  return {
+    type: 'bar',
+    options: {
+      animation: {
+        duration: 0
+      },
+      maintainAspectRatio: false,
+      plugins: {
+        tooltips: {
+          callbacks: {
```

Comment:
empty object as callbacks? This doesn't look right.

Reply:
I think it is an user config that was kept from the past. So I removed.